src/build/build-workflow.ts

```diff
@@ -1,15 +1,18 @@
 import { Task } from '..';
 import { Component } from '../component';
 import { GitHub, GithubWorkflow, GitIdentity } from '../github';
-import { BUILD_ARTIFACT_NAME, DEFAULT_GITHUB_ACTIONS_USER, setGitIdentityStep } from '../github/constants';
+import { BUILD_ARTIFACT_NAME, DEFAULT_GITHUB_ACTIONS_USER } from '../github/constants';
+import { WorkflowActions } from '../github/workflow-actions';
 import { Job, JobPermission, JobStep } from '../github/workflows-model';
 import { Project } from '../project';
 
 const BRANCH_REF = '${{ github.event.pull_request.head.ref }}';
 const REPO_REF = '${{ github.event.pull_request.head.repo.full_name }}';
-const PRIMARY_JOB_ID = 'build';
+const BUILD_JOBID = 'build';
 const SELF_MUTATION_STEP = 'self_mutation';
-const SELF_MUTATION_COMMIT = 'self_mutation_commit';
+const SELF_MUTATION_HAPPENED_OUTPUT = 'self_mutation_happened';
+const IS_FORK = 'github.event.pull_request.head.repo.full_name != github.repository';
+const NOT_FORK = 'github.event.pull_request.head.repo.full_name == github.repository';
```

Comment:
I think that should work...

```suggestion
const NOT_FORK = `!(${IS_FORK})`;
```

Reply:
OK